### PR TITLE
[FIX] ValueError: max() arg is an empty sequence

### PR DESCRIPTION
**Pull request by OpenCode**
Fix error in issue #163

### CODE REVIEW

File: stock_picking_reorder_lines/stock.py [hunk 1]
```diff
@@ -46,9 +46,14 @@ class StockPicking(models.Model):
     @api.depends('move_lines')
     def _get_max_line_sequence(self):
         for picking in self:
+            if picking.move_lines:
+                max_line_sequence = max(
+                    picking.mapped('move_lines.sequence')) + 10
+            else:
+                max_line_sequence = 0
```
Comment (pedrobaeza) [inline stock_picking_reorder_lines/stock.py]: Shouldn't this be 10?

Comment (OpenCode) [inline stock_picking_reorder_lines/stock.py]: Sure! Done! :)

**Comment by andrea4ever** [comment]
:+1:

**Comment by OpenCode** [comment]
@pedrobaeza Done! Thank you.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by giudalo** [comment]
:+1:

**Comment by tafaRU** [comment]
LGTM :+1: , I merge then.

**Pull request merged**